politeiawww/user/user.go

```diff
@@ -142,6 +128,24 @@ type User struct {
 	SpentProposalCredits []ProposalCredit
 }
 
+// ActiveIdentity returns a user's active identity. A user will always have an
+// active identity.
+func (u *User) ActiveIdentity() *Identity {
```

Comment:
Honestly, I think we need a panic to enforce the assumption that there is always a key. I agree it shouldn't happen so let's be big boys and panic if the assumption doesn't hold.